test.py

```diff
@@ -36,3 +36,13 @@ def test_md() -> None:
     html_md = g.generate_a_html(inp)
     assert html_md == """<!DOCTYPE html><html lang="en-CA"><head><meta charset="utf-8" /><meta name="viewport" content="width=device-width, initial-scale=1" /><title>Silver Blaze</title></head><body><h1>Silver Blaze</h1><p>
 I am afraid, Watson, that I shall have to go</p></body></html>"""
+
+def test_generate_a_HtmlFile_from_mdFile()->None:
+    inp=Path("./tests/inputs/example1.md")
+    output = Path("../TextHTMLPress/dist")
+    expectedOutputFile= Path("../TextHTMLPress/dist/example1.html")
+    stylesheet = ""
+    lang = "fr"
+    g=Generator(inp,output,stylesheet,lang)
+    g.parse_markdown(inp)
+    assert expectedOutputFile.exists()
```

Comment:
The above line might be changed something as """<!DOCTYPE html><html lang="en-CA"><head><meta charset="utf-8" /><meta name="viewport" content="width=device-width, initial-scale=1" /><meta name="author" content="Somebody" /><title>Zen</title></head><body><h3>Somebody</h3><h3>2021-10-15</h3><h1>Zen</h1>
<h2>Section 1</h2>
<p>Simple is better than complex.</p>
<p>Although practicality beats purity.</p>
<h2>Section 2</h2>
<p>Need more practice.</p></body></html>"""

Reply:
It is so strange when I test it local and all tests pass

Reply:
I got it, I forgot to generate wrapper in the test